contents/en/about.md

```diff
@@ -60,4 +60,5 @@ careers:
     link: https://www.kyoto-art.ac.jp/en/
 ---
 
-I'm a Product Designer with 10 years of experience working closely with startups to create and foster brands through end to end experience design. In my spare time, I like playing video game, watching anime and drinking green tea.
+I am a Brand & Product Designer with ten years of experience based in San Francisco, currently working at All Turtles, a technology company that develops AI products. My current focus is on creating and fostering brands through visual and interaction design. In the past, I worked at a creative agency and have worked on making promotional content for companies such as Google Japan, Niantic, and PlayStation.
+To explore new inspirations, I’m working on a side project called Daily Pattern, which involves creating seamless patterns. In my spare time, I play video games, watch anime, and drink green tea.
```

Comment:
```suggestion

To explore new inspirations, I’m working on a side project called Daily Pattern, which involves creating seamless patterns. In my spare time, I play video games, watch anime, and drink green tea.
```

Looks readable if we have this line as the second paragraph :)

![image](https://user-images.githubusercontent.com/85887/65261818-eb211a00-dabd-11e9-9a17-1cc266f11f29.png)